ietf-te-path-computation.yang

```diff
@@ -392,85 +387,92 @@ module ietf-te-path-computation {
     }
     leaf upper-bound {
       type uint64;
-      description "Upper bound on SVEC metric";
+      description
+        "Upper bound on SVEC metric";
     }
-  }   // grouping svec-metrics-bounds
+  } // grouping svec-metrics-bounds
 
   grouping svec-metrics-optimization {
     description
       "TE path metric bounds grouping for computing a set of
        synchronized requests";
-
     leaf metric-type {
       type identityref {
         base svec-metric-type;
       }
-      description "TE path metric type usable for computing a set of
+      description
+        "TE path metric type usable for computing a set of
          synchronized requests";
     }
     leaf weight {
       type uint8;
-      description "Metric normalization weight";
+      description
+        "Metric normalization weight";
     }
-  }   // grouping svec-metrics-optimization
+  } // grouping svec-metrics-optimization
 
   grouping svec-exclude {
-    description "List of resources to be excluded by all the paths
-      in the SVEC";
+    description
+      "List of resources to be excluded by all the paths
+       in the SVEC";
     container exclude-objects {
-      description "resources to be excluded";
+      description
+        "resources to be excluded";
       list excludes {
-        key index;
-        ordered-by user;
-        leaf index {
-           type uint32;
-           description "XRO subobject index";
-        }
+        key "index";
         description
-          "List of explicit route objects to always exclude
+          "List of Explicit Route Objects to always exclude
```

Comment:
If it is an ERO exclusion - does it matter about the ordering of the ERO objects? I think ERO exclusion objects can be in any order as long as path computation exclude them all. An inclusion ERO should have index and its order matters. 
For that reason I suggest to remove the index here.

Reply:
Resolved with https://github.com/rvilalta/ietf-te-path-computation/pull/84/commits/79060d84c9925bb6c06f4e4fafc92aee5cba49fc